From e2e49af80f49c625bd7b844a2e3abd7c1d8b7d6a Mon Sep 17 00:00:00 2001
From: Adrian Alonso <aalonso@freescale.com>
Date: Thu, 16 Feb 2012 18:19:46 -0600
Subject: ENGR00171079-2 mx6 add anaclk_2 io buffers enable macros

* Add ANACLK_2 input/output buffers enable macros.
  In orther to bypass anaclk_2 to pll4_audio need to
  set anaclk_2 input buffer enable bits.

Signed-off-by: Adrian Alonso <aalonso@freescale.com>
---
 arch/arm/mach-mx6/crm_regs.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-mx6/crm_regs.h b/arch/arm/mach-mx6/crm_regs.h
index 6dbdeaf..803631a 100644
--- a/arch/arm/mach-mx6/crm_regs.h
+++ b/arch/arm/mach-mx6/crm_regs.h
@@ -51,6 +51,8 @@
 #define ANATOP_LVDS_CLK1_SRC_SATA	0xB
 #define ANATOP_LVDS_CLK1_OBEN_MASK	0x400
 #define ANATOP_LVDS_CLK1_IBEN_MASK	0x1000
+#define ANATOP_LVDS_CLK2_OBEN_MASK  0x800
+#define ANATOP_LVDS_CLK2_IBEN_MASK  0x2000
 #define ANA_MISC2_BASE_ADDR		(MXC_PLL_BASE + 0x170)
 
 #define PLL_SETREG_OFFSET		0x4
-- 
1.7.9.5

